project/AutoLift.scala

```diff
@@ -25,8 +26,17 @@ object AutoLift{
           "-Yno-adapted-args",
           "-Ywarn-dead-code",
           "-Ywarn-value-discard",
-          "-Xfuture"),
+          "-Xfuture",
+          "-Ywarn-unused-import"),
```

Comment:
See if the `bench` project compiles with this here. There's a reason I moved it to each individual project. Also `sbt tut`.